blog/2026-04-02-state-of-ecosystem.md

```
@@ -0,0 +1,37 @@
+title: "State of the Ecosystem"
+slug: state-ecosystem-2026
+authors: castrojo
+tags: [announcements]
+---
+
+I thought I'd take some time to talk a bit about the world around us. I've come back from KubeCon + CloudNativeCon in Amsterdam super energized and have been taking this week off to recharge and relax. 
+
+I'll be talking about Bluefin and friends a bit later - we're gearing up for release and everything is humming along nicely. Let's talk about where we are.
+
+
+### Our target audience
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)